src/java.base/share/classes/java/time/ZoneOffset.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Shouldn't be 2025 too?

Reply:
This PR was published last year and `ZoneOffset` has not changed since then. So I think 2024 is fine